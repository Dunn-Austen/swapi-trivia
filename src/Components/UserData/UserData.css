.userdata-component {
  align-items: center;
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  color: white;
  width: 100%;
}

.users-data {
  margin-left: 10px;
}

.btns-container {
  align-items: center;
  color: #5ae273;
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  margin-right: 10px;
  width: 160px;
}

.menu-btn:hover {
  background-color: #5C5E5F;
  color: white;
}

.user-values {
  color: #5ae273;
}
