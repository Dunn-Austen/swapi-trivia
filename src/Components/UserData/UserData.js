import React from 'react';
import './UserData.css';

const UserData = () => {

  return (
    <header className="userdata-component">
      <div className="users-data">
        <p className="name-value">Props-name</p>
        <p className="quotation-value">Props-quotation</p>
        <p className="selected-ranking">Props-ranking</p>
      </div>
      <div className="btns-container">
        <button className="logout-btn" onClick={() => }>Log Out</button>
        <button className="show__favorites--btn" onClick={() => }>Show Favorites</button>
      </div>
    </header>
  )
};

export default UserData;
