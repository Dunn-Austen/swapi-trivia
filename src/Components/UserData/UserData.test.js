import React from 'react';
import { shallow } from enzyme

import UserData from '../UserData/UserData';


describe('UserData', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<CharacterCard
            name="Foster"
            quote="hello"
            ranking="first"
           
        />)
    })

    it.only('should match the snapshot with all data passed correctly', () => {
        expect(wrapper).toMatchSnapshot()
    })

});