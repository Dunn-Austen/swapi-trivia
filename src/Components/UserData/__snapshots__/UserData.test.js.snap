// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserData should match the snapshot with all data passed correctly 1`] = `
<nav
  className="userdata-component"
>
  <div
    className="users-data"
  >
    <p
      className="name-value"
    >
      Username: 
      <span
        className="user-values"
      >
        Foster
      </span>
    </p>
    <p
      className="quotation-value"
    >
      Quotation: 
      <span
        className="user-values"
      >
        hello
      </span>
    </p>
    <p
      className="selected-ranking"
    >
      Fandom: 
      <span
        className="user-values"
      >
        first
      </span>
    </p>
  </div>
  <div
    className="btns-container"
  >
    <button
      className="log-out"
      id="logout-btn"
      onClick={[Function]}
    >
      Log Out
    </button>
    <button
      className="menu-btn"
      id="show__favorites--btn"
    >
      Faves
    </button>
  </div>
</nav>
`;
